packages/edit-site/src/components/dataviews/README.md

```diff
@@ -5,6 +5,7 @@ This file documents the DataViews UI component, which provides an API to render
 ```js
 <DataViews
 	data={ pages }
+	getItemId={ ( item ) => item.id }
```

Comment:
I'm hesitant to make this a separate prop or a special field but this works for me. We can see over time.

Reply:
Also, should we document the prop?

Reply:
Let's do the documentation in a follow up and add more info on the other props too.